scss/grid/_flex-grid.scss

```diff
@@ -200,6 +200,29 @@
     margin-right: 0;
   }
 
+
+  .flex-container {
+    @include flex;
+  }
+
+  .flex-child-auto {
+    flex: 1 1 auto;
+  }
+
+  .flex-child-grow {
+    flex: 1 0 auto;
+  }
```

Comment:
Might need a media query for these. Like `.small-flex-child-auto` & `.medium-flex-child-grow`